clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -872,9 +872,9 @@ getTripleBasedSPIRVTransOpts(const ArgList &Args,
       ",+SPV_INTEL_fpga_argument_interfaces"
       ",+SPV_INTEL_fpga_invocation_pipelining_attributes"
       ",+SPV_INTEL_fpga_latency_control"
-      ",+SPV_INTEL_task_sequence"
```

Comment:
is this part of the change NFC or does the order matter?

Reply:
Order does seem to matter. Test fails otherwise.